Klipper_Expander/Documentation/README.md

```diff
@@ -0,0 +1,259 @@
+Welcome to your new Klipper Expander! This document outlines the common features of the board that you are likely to use. 
+
+<!-- TOC -->
+
+- [Basic](#basic)
+    - [Indicator LEDs](#indicator-leds)
+    - [Switch RST](#switch-rst)
+    - [I2C Header PF0, PF1](#i2c-header-pf0-pf1)
+    - [Power In](#power-in)
+    - [Fuse](#fuse)
+    - [Mosfets PA0, PA1, PA2, PA3](#mosfets-pa0-pa1-pa2-pa3)
+    - [Thermistors PA5, PA6](#thermistors-pa5-pa6)
+    - [Neopixel header PB1](#neopixel-header-pb1)
+- [Setup](#setup)
+- [Advanced](#advanced)
+    - [Boot Jumper](#boot-jumper)
+    - [SWD header PA13, PA14, RST](#swd-header-pa13-pa14-rst)
+    - [GPIO PA7](#gpio-pa7)
+- [Testing](#testing)
+- [Debugging](#debugging)
+    - [Basic troubleshooting:](#basic-troubleshooting)
+- [FAQ](#faq)
+    - [Why do I want this?](#why-do-i-want-this)
+    - [How do I get one?](#how-do-i-get-one)
+
+<!-- /TOC -->
+
+# Basic
+
+First we will go through the basic functionality of the board that most users will be looking for
+
+## Indicator LEDs
```

Comment:
Here i would add labels to each LED (D1,D2...) and add a 4th section for MOSFET Status LED's
- Board Ready
- USB Power (5V)
- Vin Power
- MOSFET Status

Reply:
Done!

Reply:
Looks good!